sigstore-cli/build.gradle.kts

```diff
@@ -25,6 +31,31 @@ tasks.compileJava {
 application {
     mainClass.set("dev.sigstore.cli.Sigstore")
 }
+
+tasks.named("shadowDistTar") {
+    enabled = false
+}
+
+tasks.named("shadowDistZip") {
+    enabled = false
+}
+
 tasks.run.configure {
     workingDir = rootProject.projectDir
 }
+
+tasks.register<ShadowJar>("serverShadowJar") {
+    archiveBaseName.set("sigstore-cli-server")
+    archiveClassifier.set("all")
+
```

Comment:
set archiveVersion to `""` so we don't have to wildcard in conformance.yaml:49 (I realize we were doing it before) but be nice to fix that now.